docs/data-modeling-principles.md

```diff
@@ -0,0 +1,115 @@
+it would be very challenging to cooperate across the organization, a situation which we refer to as *siloing*.
+If Solution Data Models are allowed to build on top of other Solution Data Models, it increases the complexity
+which makes it hard to maintain and understand. In addition, it can easily lead ot siloing through obfuscation.
+By using the Enterprise Data Model for shared concepts, it enforces a discussion when new concepts are introduced.
+The discussion will ensure that the new concepts are well anchored in the organization and can be built upon by other
+business areas.
+
+### All data models are kept in its own `Space`
+
+This follows from the principle of parsimonious model. By keeping each data model in its own `Space`, it is clear
+which concepts are part of the data model and which are not. It also makes it easier to manage access control,
+which should be with the team that owns the data model.
+
+### All `View`s of a data models have the same version and `Space` as the data model
+
+This follows from the principle of parsimonious model. Technically, a data model can contain `View`s from
+different `Space`s and with different version. However, `View`s are very cheap and having `View`s with different
+versions than the data model can cause a lot of confusion. Furthermore, all `View`s should be controlled by the
+team that owns the data model, thus they should be in the same `Space`. If a solution models needs to use a `View`
+that is identical to a `View` in the Enterprise Data Model, you should use the `implements` option to reference the
+`View` in the Enterprise Data Model. An advantage of this is that if you want to extend the `View` later
+in the Solution Data Model, you can do that without affecting the `View` in the Enterprise Data Model.
+
+### Keep the Enterprise Data Model as small as possible, but not smaller
+
+This follows from the principle of cooperation and parsimonious model. There is a cost to each concept in the
+Enterprise data model as it needs to be discussed and anchored in the organization. Thus, concepts that are not
+shared should not be put through this process. A side note is that concepts that are not shared can easily
+elicit strong opinions without grounding in reality, and thus be an extra long cumbersome waste of time.
+
+In addition, the organization will constantly evolve the Enterprise Data Model, and the goal is to avoid
+introducing changes that will lead to a chain of changes to all Solution Data Models. If we avoid introducing
+a new concept too early, it gives us the maximum flexibility in the future when we need to introduce the new concept.
+If the concept is already in the Enterprise Data Model, it will be much more costly to change it.
+
+Note that an organization that is mature when it comes to data modeling should expect to have three versions of the
+Enterprise Data Model: `legacy`, `current` and `next`. The `current` is the version that most solution models
```

Comment:
should we call `next` `WIP` ? of `future` ?

Reply:
I like future then it is clear that it is a timeline